src/renderer/components/QrErrorMessage.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+
+import useTranslationFunction from '../hooks/useTranslationFunction'
+
+type Props = {
+  url: string
+}
+
+export default function QrErrorMessage({ url }: Props) {
```

Comment:
This became it's own little component as passing in JSX as an `message` value into an alert dialog component doesn't work within hooks. In any case, it should be cleaner like this now.

Reply:
maybe we could group the qr code related components into a qr folder

Reply:
I'll do this as part of the QR code refactor!